### ISSUE TITLE
Relaxation not included in my version of Oceananigans.Forcing 

### ISSUE THREAD

**Issue by gregbritten** [gregbritten/TurbulentBlooms.jl#4]
I want to run `windy_convection_with_surface_waves.jl` but it requires `Relaxation` which is not included when I add `Oceananigans.jl`

**Comment by glwagner** [comment]
Are you starting Julia with 

julia --project 

?

**Comment by glwagner** [comment]
This failed for me too at first, but was fixed by updating `Oceananigans` to master. (In the current environment, `Oceananigans` is tagged to #glw/better-hascuda) in an attempt to address #3.)

Can you paste the output of

```
julia --project -e 'using Pkg; Pkg.status()'
```

run from the root of this repository?

**Comment by gregbritten** [comment]
OK you'll have to pardon my unfortunate windows laptop situation here... 
Julia has not worked particularly well for me from Cygwin so I've been working from Atom/Juno

I believe the equivalent of 

`julia --project -e`

in Atom is to issue the following from the root of the repo

`(@v1.4) pkg> activate .`

which gives 

`Activating environment at D:\Dropbox\Working\turbulent_blooms\github\Project.toml`

Then I issue:

```
julia> Pkg.instantiate()
julia> Pkg.status()
```

which gives
```
Status `D:\Dropbox\Working\turbulent_blooms\github\Project.toml`
  [0d21a966] LESbrary v0.1.0 #master (https://github.com/ali-ramadhan/LESbrary.jl.git)
  [9e8cae18] Oceananigans v0.30.0 #master (https://github.com/CliMA/Oceananigans.jl.git)
  [de0858da] Printf
  [9a3f8284] Random
  [10745b16] Statistics
```

**Comment by glwagner** [comment]
Ok, hopefully this issue is fixed by #5.
